content/community/irc/_index.html

```diff
@@ -10,6 +10,8 @@ <h3>Chat channels</h3>
 <p>Haiku has chat channels in various languages, allowing realtime discussion
 with other Haiku developers and users.</p>
 
+<p><b>NOTE:</b> We are currently in the process of transitioning most of our channels from the Freedom IRC network to the OFTC IRC network.</p>
```

Comment:
I think we have a dedicated css class to make a nice alert box (with icon and everything).